### PR TITLE
QA-14375: backport fix and fix dependencies

### PR DESCRIPTION
**Pull request by cedmail**
## JIRA

https://jira.jahia.org/browse/QA-14375

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -184,6 +184,106 @@
...
+      <artifactId>transaction-api</artifactId>
+      <version>1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>antlr</groupId>
+      <artifactId>antlr</artifactId>
+      <version>2.7.6</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.tiles</groupId>
```
Comment (gflores-jahia) [inline pom.xml:238]: duplicate entries on some of the org.apache.tiles dependencies

**Comment by gflores-jahia** [comment]
just curious, but where are these new additional entries coming from? (I didn't see it from the 3.2.x branch of source https://github.com/spring-projects/spring-framework/tree/3.2.x)

**Comment by cedmail** [comment]
> just curious, but where are these new additional entries coming from? (I didn't see it from the 3.2.x branch of source https://github.com/spring-projects/spring-framework/tree/3.2.x)

I have updated the README last time to explain that the pom.xml is here to download the dependencies only, as JDK 1.7/Gradle 2.5 is not able to downlaod most of the dependencies anymore

**Comment by gflores-jahia** [comment]
Thanks I see it now coming from https://github.com/spring-projects/spring-framework/blob/3.2.x/build.gradle

**Review by gflores-jahia — APPROVED** [review]

**Pull request merged**